src/renderer/template.ts

```diff
@@ -48,7 +47,7 @@ function renderFile(file: React.ReactElement): TemplateRenderResult | undefined
   const { type, props = {} } = file;
 
   // if no File component is found as root, don't render it.
-  if (type !== File) {
+  if (typeof type !== "function" || type.name !== "File") {
```

Comment:
Is it possible `type` returns `undefined`?

Reply:
Yes is possible but with `typeof type !== "function"` comparison you check that type must be function, otherwise don't go to second comparison :)

Reply:
For a sec I thought typeof would result in undefined being thrown LGTM 👍